LBHFSSPublicAPI.Tests/V1/Gateways/AddressesGatewayTests.cs

```diff
@@ -16,21 +16,21 @@ namespace LBHFSSPublicAPI.Tests.V1.Gateways
     public class AddressesGatewayTests
     {
         private IAddressesGateway _classUnderTest;
-        private Mock<IAddressesAPIContext> _mockAddressesAPIContext;
+        private Mock<IAddressesAPIContext> _mockAddressesApiContext;
 
         [SetUp]
         public void Setup()
         {
-            _mockAddressesAPIContext = new Mock<IAddressesAPIContext>();
-            _classUnderTest = new AddressesGateway(_mockAddressesAPIContext.Object);
+            _mockAddressesApiContext = new Mock<IAddressesAPIContext>();
```

Comment:
I think it was as it was due to API being an acronym, but I guess it's alright this way as well.

Reply:
Yeah I changed it because my IDE was complaining.  It didn't know it was an acronym.